packages/bruno-app/src/components/RequestPane/HttpRequestPane/index.js

```diff
@@ -24,6 +24,14 @@ const ContentIndicator = () => {
   );
 };
 
+const ErrorContentIndicator = () => {
```

Comment:
I don't think ErrorContentIndicator makes sense.
"ContentIndicator" suggests there's content, and adding "Error" to it makes it sound like there's something called "Error Content," which feels odd.

I think `ErrorIndicator` is simpler and makes more sense.